src/bepasty/utils/upload.py

```diff
@@ -128,6 +126,6 @@ def create_item(f, filename, size, content_type, content_type_hint,
 @threaded
 def background_compute_hash(storage, name):
     with storage.openwrite(name) as item:
-        size = item.meta[SIZE]
+        size = item.meta[constants.SIZE]
         file_hash = compute_hash(item.data, size)
-        item.meta[HASH] = file_hash
+        item.meta[constants.HASH] = file_hash
```

Comment:
I see how you put `constants.` to avoid the pep warning, but i think the old way was easier to read.

If there is only one module where such uppercase constants come from, I think this is considered as a valid use of a star import.

So, just add noqa to silence pep8?

Reply:
It's not only silencing pep8 warnings. It's just ugly and confusing to use `from module import *`. You should avoid that in EVERY situation, you are polluting namespace with thing you are not aware of.
So if not `from .. import constants` and not `from ..constants import *`, maybe this?
```
from ..constants import (
    FOREVER,
    FILENAME,
    HASH,
    SIZE,
    ...    
)
```

Reply:
yeah, the latter would be a good way to solve both issues. bit harder to maintain, but guess we can live with that.